src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -395,10 +395,20 @@ cdef class Font:
     cdef Face _face
     cdef FontFuncs _ffuncs
 
-    def __cinit__(self, Face face):
+    def __init__(self, face_or_font):
```

Comment:
allocation of C data structures should be done in the special `__cinit__` method, which is guaranteed to be called exactly once: 
https://cython.readthedocs.io/en/latest/src/userguide/special_methods.html#initialisation-methods-cinit-and-init 

i'll fix that on `main`

Reply:
https://github.com/harfbuzz/uharfbuzz/commit/b7a809ddedd138622ccb2d9c573a64528a16eb67